### ISSUE TITLE
!bool casts all string values to True

### ISSUE THREAD

**Issue by tommilligan** [taverntesting/tavern#149]
Tavern provides a `!bool` constructor that is used during configuration for [type conversion](https://taverntesting.github.io/documentation#type-conversions).

Unfortunately, it uses Python's underlying [`bool` constructor](https://docs.python.org/3.7/library/functions.html#bool) which casts all non-empty strings to `True`.

```yaml
value: !!bool "True"
# True
value: !bool "{v_true}"
# True

value: !!bool "False"
# False
value: !bool "{v_false}"
# True
```

[This integration test](https://github.com/taverntesting/tavern/blob/61065bd294b03a33caa0d2ef15fc2908181b5964/tests/integration/test_typetokens.tavern.yaml#L201) would fail if the value was checked.

**Comment by tommilligan** [comment]
Fixed by #150

**Comment by tommilligan** [comment]
PR #150 submitted and passed CI

**Comment by tommilligan** [comment]
For reference, my specific use case was trying to disable request SSL checking with a variable (to use in my local development environment):
```yaml
variables:
  verify-ssl: False

---

test_name: Test health
stages:
- name: health
  request:
    url: ...
    method: POST
    json:
      ...
    verify:
      value: !bool "{verify-ssl}"
```

**Comment by ke4roh** [comment]
Tom, do you have a working example of this verify value?  I have that exact problem and haven't figured out how to set it by environment variable with quite a bit of tinkering and attempting to mimic tests.

**Comment by tommilligan** [comment]
This PR was released as of `0.15.1`, so first check your version is up to date.

I don't actually have my old example in front of me, but I imagine setting with an env var would look something like:

```yaml
verify:
  value: !bool "{tavern.env_vars.YOUR_SSL_ENV_VAR_HERE}"
```

Sorry to not be more help - let me know if you have any specific traceback/debugging issues.

Environment variables: https://taverntesting.github.io/documentation.html#environment-variables
Type conversion: https://taverntesting.github.io/documentation.html#type-conversions

**Comment by swontroba** [comment]
Tom, I also have a question realted to this. I found this ans also want to use it for a local development environment. 
I defined a variables  in a .yaml include file. 
testing with with verify: false the test works fine. 
If I put verify: !bool '{v_verify-ssl}'  is get ["Value '<tavern.util.loader.BoolToken object at 0x0379BB50>' is not of type 'bool'. Path: '/stages/0/request/.....

test_name: Check if we... information

includes:
  - !include common.yaml
  - !include common_trs.yaml
  - !include common_ips.yaml

stages:  
  - name: Get information without auth fails
    request:
      url: "https://{host}:{port}/api/trs/{ip}/{trs_de}"
      method: GET
      verify: !bool '{v_verify-ssl}'

    response:
      status_code: 401
      body:
        Message: "Authorization has been denied for this request."

I don't know if I have any other formating problem - I am new to tavern/pytest and testing with it. But with just the false it works. So I assume there is something else.
I used python 3.7 and also tested 3.5. intalling everything "fresch" via pip
Thanks
